packages/plugin-vue/src/template.ts

```diff
@@ -123,7 +123,7 @@ export function resolveTemplateCompilerOptions(
           slash(path.relative(options.root, path.dirname(filename)))
       }
     }
-  } else {
+  } else if (options.resolveAbsoluteUrls) {
```

Comment:
1. I think this is strictly a `plugin-vue` concern and should not need a Vite option.
2. If we switch the spread order on line 142, we can allow user's `template.transformAssetUrls` option to overwrite the default, including `template.transformAssetUrls.includeAbsolute`. Which means we don't even need an extra option for this.

Reply:
Option 2 is fine if I can override this from another plugin, but my understanding is that I cannot, right?